test/views/behaviours/QuestionViewBehaviours.scala

```diff
@@ -243,13 +243,13 @@ trait QuestionViewBehaviours[A] extends ViewBehaviours {
         "show an error summary" in {
 
           val doc = asDocument(createView(form.withError(FormError(dateKey, "error"))))
-          assertRenderedById(doc, "error-summary-title")
+          assertRenderedByClass(doc, "error-summary-title")
```

Comment:
Change "error-summary-title" to "govuk-error-summary"

Reply:
Done